plugins/dev9ghzdrk/dev9ghzdrk.ui

```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk+" version="2.24"/>
+  <!-- interface-naming-policy project-wide -->
+  <object class="GtkAboutDialog" id="IDD_ABOUT">
```

Comment:
I don't think this is needed anymore since you removed the about box code?

Reply:
Removed